lite/tests/classification/test_accuracy.py

```diff
@@ -53,18 +53,12 @@ def test_accuracy_computation():
     )
 
     # score threshold, label, count metric
-    assert accuracy.shape == (2, 4)
+    assert accuracy.shape == (2,)
```

Comment:
```suggestion
    assert accuracy.shape == (2)
```
delete the trailing comma

Reply:
The comma is actually correct here 😅 

`(2)` evaluates to `2` of type `int`
`(2,)` evaluates to `(2,)` of type `tuple[int]`